packages/core/src/sync/events.ts

```diff
@@ -423,6 +564,71 @@ export const decodeEvents = (
         break;
       }
 
+      case "account": {
+        switch (source.filter.type) {
+          case "transaction": {
+            // TODO(kyle) what if toAddress and fromAddress are both undefined?
```

Comment:
Not sure what you mean here

Reply:
How do we know where do display `{sourceName}:transaction:from` or `${sourceName}:transaction:to` if the user creates an account with `{ address: undefined }`. I think we should actually validate against this